### DISCUSSION TITLE
PSX/PS2 Adapter Assembly

### DISCUSSION THREAD

**Discussion by alastairbdoggett** [darthcloud/BlueRetro#87] [Q&A] [answered]
Following the guide, I am confident I have my wiring done right. But when I plug it in and boot up the PS2 the red LED on the ESP32 is dim and I cannot get BlueRetro to connect in the Web Config.

I'm thinking something is up with the power and not enough is getting through to the ESP32 somehow since the light is dim. 

Currently, I am just using the 8V on the PS controller and connecting that to the 5V on the DB25 (so 2x PS 8V connected to a single 5V) -- that was I how i interpreted the below:
<img width="478" alt="Screen Shot 2021-10-28 at 9 25 00 PM" src="https://user-images.githubusercontent.com/17734670/139375453-f71c1fdd-62b5-4e37-a836-39e2ea1b21c7.png">

Am I doing something wrong? I feel like i'm missing something simple 😬

**Comment by darthcloud** [comment]
Try wiring only one of the 8V?

What ESP32 board are you using?

**Reply by alastairbdoggett** [reply]
Ok will try the single wire and let you know if anything changes!

The board I am using is ESP32 
-WROOM 
-32D DevKitC
![01236A3A-748C-4FD1-954A-15F118C3B269](https://user-images.githubusercontent.com/17734670/139599819-b0b4207e-9c4d-4858-8ebc-24aa5a7c1bff.jpeg)

**Reply by darthcloud** [reply]
Confirm you got 8V by using a multimeter between GND and 8v pins.

**Reply by alastairbdoggett** [reply]
To clarify: you mean connect multimeter test leads to 8V pin and GND pin of the PSX/PS2 controller pinout when console is turned on and PSX/PS2 controller is plugged into console?

**Reply by darthcloud** [reply]
Yes but nothing connected besides your cable

**Reply by alastairbdoggett** [reply]
Confirming I get 8V using the multimeter (picture below). Also tried wiring only one of the 8V but still getting the same issue (not found in Web Config and dimmer LED than USB power).

![0B1B6AB4-8717-408F-BE67-242C38EBB094](https://user-images.githubusercontent.com/17734670/139621375-8a9b9722-f5de-4c29-89f5-9bc237acdffa.jpeg)

**Reply by darthcloud** [reply]
Ok, you validated the 8v endup on the esp32 5V input?

Is the regulator on the devkitc a AMS1117-3.3? (the 4 pins transitor), You could have a board with a fake one too.

Maybe try to buy the same board from digikey.

**Reply by alastairbdoggett** [reply]
Yes have confirmed that 8V is also present at the ESP32 5V pin. 

I have a few other boards lying around (not 32D DevKitC, but other variants of of ESP32 WROOM 32x) so I will try flashing these and trying again. I’ll also grab another 32D DevKitC board as you suggested :)

**Comment by darthcloud** [comment]
Remove the 8V connection and power from usb cable, does it work?

**Reply by alastairbdoggett** [reply]
No this doesn't work either.

I have also tried another dev board and still no luck. All the boards I have tried do work with my GC adapter though!

**Reply by darthcloud** [reply]
If it doesn't work with USB power, you got wiring issue for sure. Try FW that is hardcoded for PSX to rule out detection issue.

**Reply by alastairbdoggett** [reply]
It doesn't work with FW that is hardcoded for PSX. I'm guessing i've made a critical error somewhere with my wiring and/or connections. I'm going to tear it down and build it up again carefully :)

Thank you for the debug stps/suggestions!

**Reply by alastairbdoggett** [reply]
Not sure if this is relevant, but I’m using PS2 controller extension cables (like the ones in the picture below) in my build rather than PS2 controllers

![170B0814-3EB0-4D0F-8F4F-B50A1B09B318](https://user-images.githubusercontent.com/17734670/139944513-b01b96a8-1d5a-449e-824e-949544633cdb.jpeg)

My intuition says this shouldn’t affect anything but thought I’d check just in case this might be causing the issue!

**Reply by alastairbdoggett** [reply]
Update: using 3.5V pin instead of 8V on PS2 controller pinout to connect to 5V pin 7 on DB25 gives me a FULL LED light as opposed to a dim light that I was getting with the 8V -- but I am still not able to find it in the Web Config.

Do you think this implies the onboard LDO (AMS1117-3.3) is faulty/fake -- I know you mentioned this before. I have tried this with two boards, so I think the probability that both are faulty is likely to be low.

**Reply by darthcloud** [reply]
Yeah extension is what everyone use, don't break real controller!!

If they are from same manufacturing batch they likely same source.

Maybe focus on getting it work with usb power, this has the benefit of outputting logs, see https://github.com/darthcloud/BlueRetro/wiki/Getting-BlueRetro-debug-logs-via-Serial-port-Windows-10

Get the log so we can see what's going on.

**Reply by alastairbdoggett** [reply]
Here are the logs for the PSX/PS2 adapter. I turn on the console, pair a PS4 controller, try and use it, then turn off the console.

[putty_PS2.log](https://github.com/darthcloud/BlueRetro/files/7465098/putty_PS2.log)

**Reply by alastairbdoggett** [reply]
For reference, here are the logs for the GC adapter which works (I perform the same actions as I did for PS2).

[putty_GC.log](https://github.com/darthcloud/BlueRetro/files/7465104/putty_GC.log)

**Comment by alastairbdoggett** [comment]
I have been doing a little more experimentation today and thought i summarize my findings:

The majority of the time, this is the observed behavior:

1. When console is first turned ON, I get:
```
GPIO[19]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[21]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[22]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[25]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[32]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[33]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[34]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[35]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[39]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
Detected system : 20: Exp Board
```

2. Then, when trying to pair a PS4 controller, some error seems to occur:
```
Guru Meditation Error: Core  0 panic'ed (LoadProhibited). Exception was unhandled.
...
================= CORE DUMP START =================
...
================= CORE DUMP END ===============
```
3. System then reboots and seems to successfully detect the PS2
```
Rebooting...
���������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������[0;32mI (235) cpu_start: Pro cpu up.[0m
[0;32mI (236) cpu_start: Single core mode[0m
[0;32mI (243) cpu_start: Pro cpu start user code[0m
[0;32mI (244) cpu_start: cpu freq: 240000000[0m
[0;32mI (244) cpu_start: Application information:[0m
[0;32mI (244) cpu_start: Project name:     BlueRetro[0m
[0;32mI (244) cpu_start: App version:      v1.1 universal_spiffs[0m
[0;32mI (245) cpu_start: Compile time:     Oct 21 2021 00:46:14[0m
[0;32mI (246) cpu_start: ELF file SHA256:  fbd7234d979c9fe0...[0m
[0;32mI (247) cpu_start: ESP-IDF:          v4.4-dev-2128-gf45b60b28b[0m
[0;32mI (248) cpu_start: App cpu up.[0m
[0;32mI (248) heap_init: Initializing. RAM available for dynamic allocation:[0m
[0;32mI (249) heap_init: At 3FF80000 len 00002000 (8 KiB): RTCRAM[0m
[0;32mI (250) heap_init: At 3FFAFF10 len 000000F0 (0 KiB): DRAM[0m
[0;32mI (250) heap_init: At 3FFB6388 len 00001C78 (7 KiB): DRAM[0m
[0;32mI (251) heap_init: At 3FFB9A20 len 00004108 (16 KiB): DRAM[0m
[0;32mI (252) heap_init: At 3FFD6A58 len 000095A8 (37 KiB): DRAM[0m
[0;32mI (253) heap_init: At 3FFE0440 len 00003AE0 (14 KiB): D/IRAM[0m
[0;32mI (254) heap_init: At 3FFE4350 len 0001BCB0 (111 KiB): D/IRAM[0m
[0;32mI (254) heap_init: At 40078000 len 00008000 (32 KiB): IRAM[0m
[0;32mI (255) heap_init: At 4009D638 len 000029C8 (10 KiB): IRAM[0m
[0;32mI (257) spi_flash: detected chip: generic[0m
[0;32mI (257) spi_flash: flash io: dio[0m
[0;32mI (258) esp_core_dump_uart: Init core dump to UART[0m
[0;32mI (258) esp_core_dump_uart: Init core dump to UART[0m
[0;32mI (259) cpu_start: Starting scheduler on PRO CPU.[0m
App main at 40082F80
# bt_host_load_bdaddr_from_file: No BDADDR on SD. Using ESP32's MAC
[0;32mI (325) BTDM_INIT: BT controller compile version [d874f68][0m
[0;32mI (325) system_api: Base MAC address is not set[0m
[0;32mI (325) system_api: read default base MAC address from EFUSE[0m
[0;32mI (325) phy_init: phy_version 4670,719f9f6,Feb 18 2021,17:07:07[0m
# bt_hci_cmd_reset
[0;32mI (735) gpio: GPIO[0]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0 [0m
# BT_HCI_EVT_CMD_COMPLETE
# bt_hci_cmd_read_local_features
App init at 40082F90
APP_CPU RESET: 0
APP_CPU CLKGATE: 0
APP_CPU STALL: 0
APP_CACHE_CTRL: 00000030
APP CPU STACK PTR: 3FFE0E98
GPIO[19]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[21]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[22]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[25]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[32]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[33]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[34]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[35]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[39]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
# BT_HCI_EVT_CMD_COMPLETE
# bt_hci_cmd_read_local_version_info
# BT_HCI_EVT_CMD_COMPLETE
# bt_hci_cmd_read_bd_addr
# BT_HCI_EVT_CMD_COMPLETE
# local_bdaddr: 7C:9E:BD:4B:96:0A
# bt_hci_cmd_read_buffer_size
GPIO[19]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[21]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[22]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[25]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[32]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[33]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[34]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[35]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[39]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
# Detected system : 17: PS2
GPIO[2]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0
GPIO[34]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:1
GPIO[5]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:1
```

4. Re-pairing the PS4 controller maintains a stable connection but does not have any response from the console (i.e. cannot navigate system menu etc.).

**Reply by alastairbdoggett** [reply]
Occasionally, Step 3 changes and instead of it recognizing PS2 console it will recognize SNES:
```
GPIO[19]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[21]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[22]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[25]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[32]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[33]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[34]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[35]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:3
GPIO[39]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[19]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[21]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[22]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[25]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[32]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[33]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[34]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[35]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[39]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
# Detected system : 6: SNES
GPIO[32]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:1
GPIO[5]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:2
GPIO[18]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:2
GPIO[19]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[21]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[22]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[25]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
```

**Reply by alastairbdoggett** [reply]
Update: Based on the behavior I saw above, I decided to double-check my wiring between DB25 and ESP32 -- I did find one wrong connection and upon fixing I now consistently get:

```
GPIO[21]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[22]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[25]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[32]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[33]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[34]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[35]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[39]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
# Detected system : 13: PC-FX
GPIO[33]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:2
GPIO[5]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
GPIO[26]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:2
GPIO[18]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0
```

I now also get stable connection with PS4 controller (no error dump). However, still no response from console.

I believe this is because proper system detection is not occuring -- it is detecting `13: PC-FX` instead of `17: PS2`!

**Reply by darthcloud** [reply]
Use the hardcoded PSX FW, if that work then you got some detection pin floating.

If hardcoded dont work, wiring is not ok.

**Reply by alastairbdoggett** [reply]
yeah the PSX FW doesn't work 😕 

the logs show a lot of repeated lines:
```
00: Unk cmd: 0xC5
00: Unk cmd: 0xC2
```

i'm pretty stumped -- have checked continuity all pinout of P1 and P2 controller cable

**Comment by darthcloud** [comment]
so PSX pin 6 is connected on I34 on esp32? For detecting PC-FX that would means I34 is  not properly connected.

**Reply by alastairbdoggett** [reply]
yeah I have:

```
PSX pin 6 -> DB25 pin 10 -> ESP32 IO34
```

is that correct?

**Reply by darthcloud** [reply]
Im out of idea, in any case its best to get PSX FW working first.

Maybe disable 2nd port by connecting io5, io26, io27 to 3.3v

**Reply by alastairbdoggett** [reply]
No problem, really appreciate all the time, tips and suggestions! Will attempt to simply get PSX FW working :)

I will also try some different extension cables

**Reply by alastairbdoggett** [reply]
I have attempted  PSX FW specific build (13 wires total: 2x GND, 1x 8V, 10x DATA connected directly to ESP32 board). 

With USB power, controller pairs but no response from console (logs look OK, no errors). Without USB power, device not found in Web Config and no pairing. 

All pins have 3.5V except for pin 3 which has 8V when checked with multimeter (not sure if this is relevant/useful)

I have also tried different extension cables but the same problems persist.

Is it possible my PS2 might not be compatible with BlueRetro?!

**Reply by darthcloud** [reply]
I think it's unlikely, next step would be to make it single plug. Also check continuity of each connection with multimeter "beep mode". Very often cable are broken inside the sleeve. Keep all cable <= 1 feet max.

But yeah if you can try another PS2 or a PS1 that would help.

**Reply by alastairbdoggett** [reply]
Testing voltage at the pins of ESP32 I noticed something strange:

When PSX 8V (pin 3) is **not** connected to ESP32 it reads 8V as expected. However, as soon as it is connected to 5V pin on ESP32 it reads 1.8V. Is this expected behavior? It may explain why the red LED is so dim since it appears it is only receiving 1.8V.

I have replaced all wires with 1" ones and tested continuity and all is OK

**Reply by darthcloud** [reply]
Not normal, but again you should unconnected 8V and run with USB power until you figure the fault.

**Comment by alastairbdoggett** [comment]
When you say:

```
Keep all cable <= 1 feet max.
```

Are you referring the the PS2 extension cables?!

**Reply by darthcloud** [reply]
Yes, At this point just cut the cord and connect directly the 7 wire on the ESP32.

**Reply by alastairbdoggett** [reply]
Ah -- I have been using extension cables >1ft this whole time 😳

**Reply by darthcloud** [reply]
Could be the problem, but at the same time that wouldn't give you 8v issues.

**Reply by alastairbdoggett** [reply]
Cutting extension cables <1 ft, connecting directly the 7 wire on ESP32 and using USB power I can now get the PS4 controller to pair and work on the console! (PSX FW specific)

**Reply by alastairbdoggett** [reply]
The issue with 8V still remains though (as you correctly predicted!). 

Here are some observations:
- When only 8V wire connected directly to ESP32, the 5V pin on ESP32 receives the expected 8V ✅ 
- When 8V and 5x DATA are connected directly to ESP32, the 5V pin on ESP32 receives ~4V
- When 8V and 5x DATA and GND are connected directly to ESP32, the 5V pin on ESP32 receives ~2V

**Reply by darthcloud** [reply]
I think the power regulator of your board cant handle 8V.

Glad you got it working somehow! 👍

**Reply by alastairbdoggett** [reply]
Would you suggest grabbing a few other ESP32 boards from different sources to see ifI happen upon one has a power regulator that can handle it? Digikey is out of stock currently.

Or do you have another ESP32 supplier that you know can handle the PS2 8V?

**Comment by darthcloud** [comment]
The thing is official one are out of stock and with the IC shortage worldwide I feel other source are likely putting whatever. Possible you get a better one, maybe not.

**Reply by alastairbdoggett** [reply]
Got it, makes sense! Again, thank you for being patient and helping me troubleshoot, it's much appreciated and I learnt a lot ☺️

**Comment by alastairbdoggett** [comment, marked as answer]
@darthcloud just an update: i managed to get some official boards from digikey ([ESP32-DEVKITC-32E](https://www.digikey.com/en/products/detail/ESP32-DEVKITC-32E/1965-ESP32-DEVKITC-32E-ND/12091810)) but the power issue is still persisting (i.e. need USB power for BlueRetro to work, rather than 8V from the PSX adapter)

**Reply by darthcloud** [reply]
This affect only a specific console? Like is PS2 or another PSX fine? 

Ultimately you could try power it from 3.3v pin on psx connector to esp 3.3v directly and see if that work.

**Reply by alastairbdoggett** [reply]
i have only tried this with a single PS2 console (i don't have any others unfortunately!)

ah that's a good idea! i will try 3.3v psx pin to esp 3.3v direct power and see if that works

**Reply by alastairbdoggett** [reply]
@darthcloud this fixed the problem! 🎉

**Reply by darthcloud** [reply]
ok weird maybe your console got 8V supply issue but good you got it working somehow still! :)
